packages/xstate-react/test/useMachine.test.tsx

```diff
@@ -647,4 +647,57 @@ describe('useMachine (strict mode)', () => {
     );
     done();
   });
+
+  // https://github.com/davidkpiano/xstate/issues/1334
+  it('should work with a rehydrated state', (done) => {
```

Comment:
Maybe something like this?

```suggestion
  it('delayed transitions should work when initializing from a rehydrated state', (done) => {
```